File: evidence/generated-systems/angular/insurance-on-angular/src/app/models/Underwriter.js

// Define collection and schema for Underwriter
export interface Underwriter {
    firstName:
	type : string
    lastName:
	type : string
    employeeId:
	type : string
    authorityLimit:
	type : Money
    Decisions:
 	type : [{ type: Schema.Types.ObjectId, ref: 'UnderwritingDecision' }]
    Insurer:
	type : Schema.Types.ObjectId
#
    collection: 'underwriters'
}

File: evidence/generated-systems/rubyonrails/insurance-on-rails/insuranceonrails/app/models/underwriter.rb
class Underwriter < ApplicationRecord


  composed_of :money,
    class_name: "Money",
    mapping: [
      ${$mapping}, 
      %w[money_currency currency]
    ]

  has_many :Decisions, class_name: 'UnderwritingDecision'
  has_many :Insurer, class_name: 'Insurer'

end
